FHEM/bindings/python/lib/helloworld/helloworld.py

```diff
@@ -32,13 +32,16 @@ async def Set(self, hash, args, argsh):
         }
         return await utils.handle_set(set_list_conf, self, hash, args, argsh)
 
-    async def set_on(self, hash, params):
-        seconds = params['seconds']
-        await fhem.readingsSingleUpdate(hash, "state", "on " + seconds, 1)
+    async def set_on(self, hash, params = None):
```

Comment:
Ich habe in der set_list_conf einen Fehler drin. Da muss noch "default": "" rein, dann braucht man muss man den Parameter nicht optional machen und erspart sich das "if" unten. Kannst du das bitte gleich in deinen PR aufnehmen?
```
"on": { "args": ["seconds"], "params": { "default": "", "seconds": {"optional": True}}},
```